### PR TITLE
Postman collection tutorial

### PR DESCRIPTION
**Pull request by nicoongit**

### CODE REVIEW

**Comment by minjikarin** [comment]
Thanks Nico! :D 
I have 2 comments 

- API keys : the API keys need to be input in 'environment' not in the individual request such as authorization. the details is here https://developers.amadeus.com/blog/how-to-play-with-self-service-apis-with-no-code-using-postman 
- at the end of page, can we also highlights that we have 'senarios' folder under 'air' and 'hotel' to illustrate better for flight booking engine flow + hotel booking engine flow. users doesn't need to copy & paste their response to request the next step.

File: docs/programming/postman.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+4. Select **Amadeus for Developers** from the **Environment** dropdown.
+
+    ![2](../images/postman-2.png)
+
+5. Give the fork a name.
+6. Select the workspace where you need to fork the collection to.
+7. Tick the **Watch original collection** box to get notified when the collection is updated.
+8. Click **Fork Collection**. You will be taken to the workspace that you selected previously.
+
+## Generate the access token
```
Comment (minjikarin) [inline docs/programming/postman.md:29]: -> APIs key to be in 'environment'

**Review by minjikarin — CHANGES_REQUESTED** [review]

**Comment by tsolakoua** [comment]
- I would suggest the authorization part to update it the same way that is in the [article](https://developers.amadeus.com/blog/how-to-play-with-self-service-apis-with-no-code-using-postman) from the sections `Configuring the environment` and `Get the access token`. What we actually do is we write small script so the access token will be passed automatically to the header of each API call. 
- Can we also show an actual API call of a specific API such as in the section `Calling Amadeus Self-Service APIs` of the blog? 
- Also, instead of showing only the JSON response, can we also show the visualization? For example if you call the Safe Place API you see also:
Screenshot 2023-02-10 at 10 45 15

**Comment by nicoongit** [comment]
Thanks @minjikarin , @tsolakoua  for the feedback. I believe I've addressed the comments, can you please check?

**Review by minjikarin — APPROVED** [review]

**Review by tsolakoua — APPROVED** [review]

**Pull request merged**